### PR TITLE
fix(macOS): Let the Metal texture be used for sampling

### LINKED ISSUE #16853: [skia][macOS] Fluent's AcrylicBrush renders pinkish instead of grey on arm64
### Current behavior

In some cases the macOS/Skia host will render scrollbar in a non-uniform pinkish color.

Screenshot 2024-05-22 at 3 12 36 PM

### Expected behavior

It should be grey.

### How to reproduce it (as minimally and precisely as possible)

Run SampleApp's `ScrollViewer_Fluent` test and look at the scrollbar color.

### Workaround

None yet

### Works on UWP/WinUI

None

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

_No response_

### Affected platforms

Skia (macOS)

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

Found while fixing https://github.com/unoplatform/uno/issues/16821

### PR DESCRIPTION
**Pull request by spouliot**
GitHub Issue (If applicable): closes #16853

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

When using Material some background have a pinkish colour instead of the expected grey.

## What is the new behavior?

The correct colour is used.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

The fact that this worked on `x86_64` really threw me off.

So it was not (now removed) the Uno `Color.cs` parsing an invalid string, nor was it something from the MS SDK, nor was it a bad RGBA to ARGB conversion inside Uno (or Skia) code.

The pink colour comes from the uninitialized texture as Metal defaults to red and some transparency was applied (over the normal grey colour). Sadly this does not report any error in the Metal API (even when special debugging/performance-affected code is used).

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m [hunk 1]
```diff
@@ -22,6 +22,7 @@ - (nonnull instancetype)initWithMetalKitView:(nonnull MTKView *)mtkView
         mtkView.colorPixelFormat = MTLPixelFormatBGRA8Unorm;
         mtkView.depthStencilPixelFormat = MTLPixelFormatDepth32Float_Stencil8;
         mtkView.sampleCount = 1;
+        mtkView.framebufferOnly = false;
```
Comment (jeromelaban) [inline src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m:25]: Why is this needed? It may be interesting to detail a bit more in a comment.

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs [hunk 2]
```diff
@@ -77,15 +76,12 @@ internal override void OnPropertyChanged2(DependencyPropertyChangedEventArgs arg
...
-	// issue specific to macOSarm64 https://github.com/unoplatform/uno/issues/16853
-	static bool macOSarm64 = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && RuntimeInformation.ProcessArchitecture is Architecture.Arm64;
-
 	private void UpdateAcrylicBrush()
 	{
 		if (_isConnected)
 		{
 			// TODO: Currently we are force recreating the brush even if it exists because Composition animations aren't implemented yet
-			CreateAcrylicBrush(useCrossFadeEffect: false, forceCreateAcrylicBrush: !macOSarm64);
+			CreateAcrylicBrush(false /* useCrossFadeEffect */, true /* forceCreateAcrylicBrush */);
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs:84]: I think named arguments are better than `/* */` comment

**Review by Youssef1313 — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs (hunk 2 above)
Comment (spouliot) [reply src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs:84]: you're right, I did a revert to keep git history cleaner - but I should reapply the naming.

File: src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m (hunk 1 above)
Comment (spouliot) [reply src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m:25]: sure, it's easy (inside Xcode) to do F1 on the member to access the header docs - but it's not something GitHub does (at least right now).

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

File: src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m (hunk 1 above)
Comment (spouliot) [reply src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOMetalViewDelegate.m:26]: comment added

File: src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs (hunk 2 above)
Comment (spouliot) [reply src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs]: updated

**Review by ajpinedam — APPROVED** [review]
